### PR TITLE
Enable the built-in help content (#1995945)

### PR DESCRIPTION
**Pull request by poncovka**
Implement the unified help support and enable the built-in help content.

**Ported from:**
* https://github.com/rhinstaller/anaconda/pull/3576
* https://github.com/rhinstaller/anaconda/pull/3575
* https://github.com/rhinstaller/anaconda/pull/3674

**TODO:**
- [x] Test with the RHEL 9 help content.
- [x] Remove the mapping file.
- [x] Update the required version of `anaconda-user-help`.

Resolves: rhbz#1995945

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
LGTM, thanks!

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks a lot! :)

File: data/help/anaconda-gui.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "_comment_": [
+    "This is a mapping of screen ids and help files for Anaconda GUI on RHEL."
```
Comment (poncovka) [inline data/help/anaconda-gui.json:3]: @M4rtinK, @VladimirSlavik, I have tested the code with this mapping file and the `anaconda-user-help-9.0.1-1` scratch build. It seems to work fine. The files `anaconda-help-anchors.json`, `rhel_help_placeholder.txt` and `rhel_help_placeholder.xml` should be removed.

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me. I just wonder the question below.

File: pyanaconda/ui/lib/help.py [hunk 2]
```diff
@@ -19,144 +19,244 @@
...
+    :param str screen_id: an identifier of a ui screen
     """
-    Return the path to the best help file for the current language and available
-    help content
+    log.info("Requested a graphical help for the '%s' screen.", screen_id)
+    help_args = _get_help_args_for_screen(DisplayModes.GUI, screen_id)
+
+    if not help_args:
+        log.debug("There is no help for the '%s' screen.", screen_id)
+        return
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/ui/lib/help.py:66]: On RHEL, we hide the button on screens without a help content (for example, the progress spoke). In upstream, no one has complained so far.

[the remark this answers is not in the document]
Comment (jkonecny12) [reply pyanaconda/ui/lib/help.py:66]: Maybe we should look on this sometimes. If no-one complains it doesn't mean that no-one is struggling with it.

However, I think this is more an upstream work.

**Review by jkonecny12 — APPROVED** [review]

**Pull request merged**